app/views/console/settings/index.phtml

```diff
@@ -473,7 +473,33 @@ $smtpEnabled = $this->getParam('smtpEnabled', false);
                                     <input name="teamId" data-ls-attrs="id=leave-teamId-{{member.$id}}" type="hidden" data-ls-bind="{{console-project.teamId}}">
                                     <input name="membershipId" data-ls-attrs="id=leave-membershipId-{{member.$id}}" type="hidden" data-ls-bind="{{member.$id}}">
 
-                                    <button class="danger">Leave</button>
+                                    <button data-ls-if="1 === {{members.memberships.length}}" class="danger" disabled>Leave</button>
+                                    <button data-ls-if="1 < {{members.memberships.length}}" class="danger">Leave</button>
+                                </form>
+
+                                <form class="pull-end" data-ls-if="{{account.$id}} === {{member.userId}}"
+                                    data-analytics
+                                    data-analytics-activity
+                                    data-analytics-event="submit"
+                                    data-analytics-category="console"
+                                    data-analytics-label="Delete Project Membership"
+                                    data-service="teams.deleteMembership"
+                                    data-scope="console"
+                                    data-event="submit"
+                                    data-success="alert,trigger,redirect"
+                                    data-confirm="Are you sure you want to remove that user from the team?"
+                                    data-success-param-alert-text="Member Removed Successfully"
+                                    data-success-param-trigger-events="teams.deleteMembership"
+                                    data-success-param-redirect-url="/console"
+                                    data-failure="alert"
+                                    data-failure-param-alert-text="Failed to Remove Member"
+                                    data-failure-param-alert-classname="error">
+
+                                    <input name="teamId" data-ls-attrs="id=leave-teamId-{{member.$id}}" type="hidden" data-ls-bind="{{console-project.teamId}}">
+                                    <input name="membershipId" data-ls-attrs="id=leave-membershipId-{{member.$id}}" type="hidden" data-ls-bind="{{member.$id}}">
+
+                                    <button data-ls-if="1 === {{members.memberships.length}}" class="danger" disabled>Leave</button>
```

Comment:
Are we okay by doing `disabled` check only client-side? In other words, is there cleanup process if the only member leaves project?

Reply:
Mentioned it in the beginning of the PR. Not sure if we want to force this behavior on all team, maybe we can add this as a feature in the future. Wouldn't block this PR for that, we can tackle it later.